spec/presenters/hyku/menu_presenter_spec.rb

```diff
@@ -89,4 +89,20 @@
       it { is_expected.to be true }
     end
   end
+
+  describe "#show_tasks?" do
+    subject { instance.show_tasks? }
+    context "for a regular user" do
+      before do
+        allow(instance.view_context).to receive(:can?).and_return(false)
```

Comment:
Probably should be specific about `receive(:can?).with(:read, :admin_dashboard)`